lib/install/management/store_files.go

```diff
@@ -189,3 +191,46 @@ func (d *Dispatcher) createVolumeStores(conf *metadata.VirtualContainerHostConfi
 	}
 	return nil
 }
+
+func (d *Dispatcher) deleteVolumeStoreIfForced(conf *metadata.VirtualContainerHostConfigSpec) {
+	if !d.force {
+		volumeStores := new(bytes.Buffer)
+		for label, url := range conf.VolumeLocations {
+			volumeStores.WriteString(fmt.Sprintf("\t%s: %s\n", label, url.Path))
+		}
+		log.Warnf("Since --force was not specified, the following volume stores will not be removed. Use the vSphere UI to delete content you do not wish to keep.\n%s", volumeStores.String())
+		return
+	}
+
+	for label, url := range conf.VolumeLocations {
+		// FIXME: url is being encoded by the portlayer incorrectly, so we have to convert url.Path to the right url.URL object
+		log.Debugf("Provided datastore URL %s", url.Path)
+		dsURL, err := vsphere.DatastoreToURL(url.Path)
+		log.Debugf("Parsed volume store path %s", dsURL.Path)
+		if err != nil {
+			log.Warnf("Didn't receive an expected volume store path format: %s", url.Path)
```

Comment:
Ah you're right, I need to add a `continue` so that the `for` loop continues to the other found volumes